docs/development/Offline_mode.md

```diff
@@ -76,3 +76,130 @@ The main problem is deciding what changes should be saved and what changes shoul
 
    
 To accommodate this there might need to add more attributes in the offline and online databases in order to deal with and keep track of the synchronization. Examples could be “last_updated_on”, “created_on”, “deleted_on”, “edited_offline” which are timestamps used to see if data should be synched or not and the “edited_offline” could be a boolean. It is also an option to use UUID with/instead of timestamps to make the synchronization have an unique id. 
+
+
+## Approaches of how to (and not to) implement the offline repository
+
+The [api_client](https://github.com/aau-giraf/api_client) maps json output from the [web-api](https://github.com/aau-giraf/web-api) into models in which the [weekplanner](https://github.com/aau-giraf/weekplanner) uses for displaying data models. Thus, it would be essential to implement the offline repository feature in the api_client. Every model in the api_client implements an abstract class called `Model` which provides a `from_json()` and `to_json()` method for the models to interact with the web-api. 

```

Comment:
```suggestion
The [api_client](https://github.com/aau-giraf/api_client) maps JSON output from the [web-api](https://github.com/aau-giraf/web-api) into models which the [weekplanner](https://github.com/aau-giraf/weekplanner) uses for displaying data models. Thus, it would be essential to implement the offline repository feature in the api_client. Every model in the api_client implements an abstract class called `Model` which provides a `from_json()` and `to_json()` method for the models to interact with the web-api. 

```